apps/tome-api/src/features/card/card.fns.ts

```diff
@@ -42,15 +49,22 @@ export const deck: DbCard[] = [
 		type: 'spell',
 		attack: 20,
 		colors: ['red', 'green'],
-		description: 'When this card is revealed, discard 1 card from your hand.',
+		description: 'When this card is revealed, discard 1 RED or GREEN (NOT IMPLEMENTED) spell from your hand.',
```

Comment:
what if they dont have any? does it just continue?

Reply:
today it'd lead to a shit experience where they can only wait until it times out. we need to check if they have a red or green card in their hand first before asking for a card

Reply:
Ideally this wouldn't even be allowed to play if you don't meet the requirements. At least how I intended the original design irl no code.

Reply:
so we need another property "canPlayCard" on cards that will be a function that returns true or false! that's actually a valid feature requirement